cvat/apps/documentation/user_guide.md

```diff
@@ -373,31 +468,96 @@ It is possible to handle many objects on the same frame in the mode.
 
 ![](static/documentation/images/image058.jpg)
 
-It is more convenient to annotate objects of the same type. For the purpose it is possible to specify a corresponding filter. For example, the following filter will hide all objects except pedestrians: ``//pedestrian``.
+It is more convenient to annotate objects of the same type. For the purpose it is possible to specify a corresponding filter. For example, the following filter will hide all objects except pedestrians: ``pedestrian``.
 
 To navigate between objects (pedestrians in the case) use the following shortcuts:
-- ``Tab`` - go to the next object
-- ``Shift+Tab`` - go to the previous object.
+- ``Tab`` — go to the next object
+- ``Shift+Tab`` — go to the previous object.
 
 By default in the mode objects are zoomed. To disable the functionality uncheck the corresponding setting: ``Open Menu`` —> ``Settings`` —> ``Zoom boxes in Attribute Annotation Mode``.
 
-By default other objects are hidden. To change the behaviour uncheck the corresponding setting: ``Open Menu`` —> ``Setting`` —> ``Hide Other in Attribute Annotation Mode``.
+By default other objects are hidden. To change the behavior uncheck the corresponding setting: ``Open Menu`` —> ``Setting`` —> ``Hide Other in Attribute Annotation Mode``.
+
+## Annotation with polygons
+
+It is used for semantic / instance segmentation.
+
+Be sure ``Z-Order`` flag in ``Create task`` dialog is enabled if you want annotate polygons.
+
+![](static/documentation/images/imag74.jpg)
+
+Before start need to be sure that ``Polygon`` is selected.
+
+![](static/documentation/images/image084.jpg)
+
+Click ``N`` for entering drawing mode. Now you can start your polygon. You can zoom in/out while drawing. Click ``N`` again for completing the shape. You can fix a position of an individual points after finishing the object. You can not add/delete points after finishing.
+
```

Comment:
Click ``N`` for entering drawing mode. Now you can start your polygon. You can zoom in/out (on mouse wheel scroll) and move (on mouse wheel press and mouse move) while drawing. Click ``N`` again for completing the shape. Also you can set fixed number of points in the field "Poly Shape Size", then drawing will be stopped automatically. You can drag object after it was drawn and fix a position of an individual points after finishing the object. You can add/delete points after finishing with help points context menu.